doc/users/colormapnorms.rst

```diff
@@ -0,0 +1,126 @@
+.. _colormapnorm-tutorial:
+
+Colormap Normaliztions 
+================================
+
+Objects that use colormaps usually linearly map the colors in the
+colormap from data values *vmin* to *vmax*.  For example::
+
+    pcm = ax.pcolormesh(x, y, Z, vmin=-1., vmax=1., cmap='RdBu_r')
+
+will map the data in *Z* linearly from -1 to +1, so *Z=0* will
+give a color at the center of the colormap *RdBu_r* (white in this
+case).  
+
+Matplotlib does this mapping in two steps, with a normalization from
+[0,1] occuring first, and then mapping onto the indices in the
+colormap. Normalizations are defined as part of
+:func:`matplotlib.colors` module.  The default normalization is
+:func:`matplotlib.colors.Normalize`.  The artists that map data to
+color pass the arguments *vmin* and *vmax* to
+:func:`matplotlib.colors.Normalize`. We can substnatiate the
+normalization and see what it returns.  In this case it returns 0.5::
+
+    norm = mpl.colors.Normalize(vmin=-1., vmax=1.)
```

Comment:
Can you either use the `ipython` directive (which will be evaluated and insert the results) or include the expected results in the text?